libraries/AP_IOMCU/iofirmware/iofirmware.cpp

```diff
@@ -711,11 +711,16 @@ void AP_IOMCU_FW::safety_update(void)
  */
 void AP_IOMCU_FW::rcout_mode_update(void)
 {
-    bool use_oneshot = (reg_setup.features & P_SETUP_FEATURES_ONESHOT) != 0;
-    if (use_oneshot && !oneshot_enabled) {
+    if ((reg_setup.features & P_SETUP_FEATURES_ONESHOT) != 0) {
+        // unused?
         oneshot_enabled = true;
         hal.rcout->set_output_mode(reg_setup.pwm_rates, AP_HAL::RCOutput::MODE_PWM_ONESHOT);
     }
+    if ((reg_setup.features & P_SETUP_FEATURES_ONESHOT125) != 0) {
```

Comment:
else if?

Reply:
Good point.
since mode is an enum, we're guaranteed only one if statement will fire, but it would save some flash space to remove the redundant tests